src/core/server/app/handlers/api/oembed/oembed.ts

```diff
@@ -46,14 +39,27 @@ export const oembedHandler = (): RequestHandler => {
         return;
       }
 
-      // Get the oEmbed response.
+      let style = `
+          body {
+            margin: 0;
+            font-family: sans-serif;
+          }
+          .container * {
+            margin: 0!important;
+          }
+      `;
       const response = await fetchOEmbedResponse(type, url, maxWidth);
       if (response === null || !response.html) {
+        const bundle = i18n.getBundle(tenant.locale);
+        const error = new EmbedNotFound().serializeExtensions(bundle);
```

Comment:
Clever solution.

I'd just recommend instead just including this translation message in the `src/core/server/locales/en-US/common.ftl` file and using the `translate` function, as we don't throw this error, and the `serializeExtensions` is typically only used in the GraphQL responses and logs.